tests/e2e/kfddistribution/furyctl-init-cluster.yaml

```
@@ -0,0 +1,150 @@
+            username: sighup
+            password: secretpassword1
+      monitoring:
+        type: mimir
+        mimir: 
+          backend: minio
+        prometheus:
+          resources:
+            requests:
+              cpu: 10m
+            limits:
+              cpu: 2000m
+              memory: 6Gi
+        minio:
+          storageSize: 20Gi
+          rootUser:
+            username: sighup
+            password: secretpassword2
+      tracing:
+        type: tempo
+        tempo: 
+          backend: minio
+        minio:
+          storageSize: 20Gi
+          rootUser:
+            username: sighup
+            password: secretpassword3
+      policy:
+        type: kyverno
+        kyverno:
+          additionalExcludedNamespaces: ["local-path-storage"]
+          installDefaultPolicies: true
+          validationFailureAction: enforce
+      dr:
+        type: on-premises
+        velero: 
+          backend: minio
+      auth:
+        provider:
+          type: none
+    # patches for kind compatibility and resource setting
+    customPatches:
+      patchesStrategicMerge:
+        - |
+          apiVersion: apps/v1
+          kind: StatefulSet
+          metadata:
+            name: minio-logging
+            namespace: logging
+          spec:
+            template:
+              spec:
+                containers:
+                - name: minio
+                  resources:
+                    requests:
+                      cpu: 10m
+                      memory: 50Mi
+        - |
+          apiVersion: apps/v1
+          kind: StatefulSet
+          metadata:
+            name: minio-monitoring
+            namespace: monitoring
+          spec:
+            template:
+              spec:
+                containers:
+                - name: minio
+                  resources:
+                    requests:
+                      cpu: 10m
+                      memory: 50Mi
+        - |
+          apiVersion: apps/v1
+          kind: StatefulSet
+          metadata:
+            name: minio-tracing
+            namespace: tracing
+          spec:
+            template:
+              spec:
+                containers:
+                - name: minio
+                  resources:
+                    requests:
+                      cpu: 10m
+                      memory: 50Mi
+        - |
+          $patch: delete
+          apiVersion: logging-extensions.banzaicloud.io/v1alpha1
+          kind: HostTailer
+          metadata:
+            name: systemd-common
+            namespace: logging
+        - |
+          $patch: delete
+          apiVersion: logging-extensions.banzaicloud.io/v1alpha1
+          kind: HostTailer
+          metadata:
+            name: systemd-etcd
+            namespace: logging
+        - |
+          $patch: delete
+          apiVersion: apps/v1
+          kind: DaemonSet
+          metadata:
+            name: x509-certificate-exporter-control-plane
+            namespace: monitoring
```

nitpick: missing new line:
```suggestion
            namespace: monitoring

```